modules/elasticsearch/src/main/java/org/testcontainers/elasticsearch/ElasticsearchContainer.java

```diff
@@ -34,15 +34,15 @@
     /**
      * Elasticsearch Default version
      */
-    protected static final String ELASTICSEARCH_DEFAULT_VERSION = "6.4.1";
```

Comment:
Unfortunately I think we have to be careful about this; anybody using the plain `ElasticsearchContainer()` constructor in their tests will get an unexpected major version bump.

We have a similar situation with other modules where the default version is getting old, and I would like to advance versions - but don't want to cause unexpected breakage.

@bsideup, @kiview let's discuss the most responsible course of action...

Reply:
Fortunately the documentation invites the user to use a specific version.
As we did not document the default constructor (which we should may be ban) I'd not be super concerned by this breaking change.

My 2 cents.